ruby/tests/common_tests.rb

```diff
@@ -814,8 +814,6 @@ def test_bad_field_names
     assert m['class'] == 2
     m['dup'] = 3
     assert m['dup'] == 3
-    m['a.b'] = 4
```

Comment:
Why removing this test?

Reply:
It's no longer possible to create a field with this name.